eisbuk-admin/src/components/slots/SlotListByDay/SlotsDay.stories.tsx

```diff
@@ -49,7 +49,7 @@ const slots = {
     categories: [Category.Course],
     type: SlotType.OffIceGym,
     durations: Object.values(Duration),
-    notes: "Pista 2\nPotrebbe non svolgersi",
+    notes: "Pista 2\n Potrebbe non svolgersi",
```

Comment:
Most nitpicky comment I ever made: it was better without the space: with this change the resulting string will have a line that starts with a space. Not worth changing though. Just mentioning in case I'm missing something.